pkg/volume/gcepd/attacher.go

```diff
@@ -313,12 +313,13 @@ func (attacher *gcePersistentDiskAttacher) MountDevice(spec *volume.Spec, device
 		return err
 	}
 
-	options := []string{}
+	options := []string{"discard"}
```

Comment:
Is this intentional change?

Reply:
yes because we want to use "discard" option at mount time for Persistent Disk and "nodiscard" when we're formatting a never-used-before disk. this is vendor-dependent

Reply:
but why just here and why just part of this PR change? I thought to fix the flake - https://github.com/kubernetes/kubernetes/issues/97071#issuecomment-739059509 we only need to use `nodiscard` ?

Reply:
ahh yes, so basically we want to use `nodiscard` for formatting to speed up the `mkfs` command, however it seems that using that is dependent on vendors. So we don't want to break other drivers. For PD specifically, it is ok to use `nodiscard` at format time for new disks, as long as the `discard` option is used at mount time. Thus the change

does that answer your question?

Reply:
can you move this to a constant such as `defaultMountOption = "discard"` and then add a comment about why we are doing this?

Reply:
done